public/static/pathways/neoadjuvant_pathway.json

```diff
@@ -240,6 +248,14 @@
       "ChemotherapyACTHP": {
         "label": "Chemotherapy ACTHP",
         "action": [
+          {
+            "type": "create",
+            "description": "Chemo Care Plan",
+            "resource": {
+                "resourceType": "CarePlan",
+                "title": "ChemoACTHP"
+            }
+          },
           {
```

Comment:
Yea we probably should remove them. If we do though can we either link to this version in the update careplan ticket or copy them over to that task (just so whoever picks it up can quickly see what the medications are instead of having to look up the regimens again)